test/image_test.py

```diff
@@ -970,6 +970,13 @@ def test_image_docker_command_entrypoint_nonempty(builder_version, servicer, cli
         assert 'ENTRYPOINT ["/temp.sh"]' in layers[0].dockerfile_commands
 
 
+def test_image_docker_command_entrypoint_malformed():
+    with pytest.raises(InvalidError, match="must be a list of strings"):
+        Image.debian_slim().entrypoint("this is not a list of strings")  # type: ignore
+    with pytest.raises(InvalidError, match="must be a list of strings"):
+        Image.debian_slim().entrypoint(4711)  # type: ignore
```

Comment:
FYI could use `pytest.mark.parameterize` ([docs](https://docs.pytest.org/en/stable/example/parametrize.html)) to reduce code repetition here. (Just a tip for the future; no need to fix).